naming/src/main/java/com/alibaba/nacos/naming/web/DistroFilter.java

```diff
@@ -124,9 +123,9 @@ public void doFilter(ServletRequest servletRequest, ServletResponse servletRespo
                 String body = IoUtils.toString(req.getInputStream(), Charsets.UTF_8.name());
 
                 HttpClient.HttpResult result =
-                    HttpClient.request("http://" + distroMapper.mapSrv(groupedServiceName) + urlString, headerList,
-                        StringUtils.isBlank(req.getQueryString()) ? HttpClient.translateParameterMap(req.getParameterMap()) : new HashMap<>(2)
-                        , body, PROXY_CONNECT_TIMEOUT, PROXY_READ_TIMEOUT, Charsets.UTF_8.name(), req.getMethod());
+                    HttpClient.request("http://" + distroMapper.mapSrv(groupedServiceName) + req.getRequestURI(), headerList,
```

Comment:
Although the application/x-www-form-urlencoded type of data can be successfully forwarded, there are some differences between it and the actual HTTP request semantics